conditional/util/email.py

```diff
@@ -0,0 +1,51 @@
+import smtplib
+from email.mime.text import MIMEText
+from email.utils import formatdate
+from email.mime.multipart import MIMEMultipart
+from datetime import date
+from conditional import app
```

Comment:
Please organise these like in the rest of conditional:

- python modules
- pip modules
- local conditional modules

Reply:
this is already the case

Reply:
Separate with a line break please